examples/pytorch/env.yml

```diff
@@ -4,14 +4,15 @@ dependencies:
   - python=3.8
   - pip
   - pip:
-    - torch==1.10.0+cu111.moreh23.6.0
-    - moreh-driver==23.6.0
+    - torch==1.13.1+cu116.moreh23.7.202
```

Comment:
Does it have to be this specific MAF version? Or it'd better stick to the latest one (which is 23.8.1)

Reply:
I haven't tried the latest version, so I think it's better to stick with this version so I can run and collect the metrics before move to the latest.